Multisig/UI/Safe Management/Edit Safe Name/EditSafeNameView.swift

```diff
@@ -35,26 +35,30 @@ struct EditSafeNameView: View {
 
     var body: some View {
         VStack(spacing: 24) {
-            RoundedTextField(title: "Enter name",
-                        text: $model.enteredText,
-                        isValid: $model.isValid,
-                        onEditingChanged: { ended in
-                            if !ended {
-                                self.model.onEditing()
-                            }
-                        },
-                        onCommit: submit)
+            VStack(alignment: .leading) {
```

Comment:
Maybe it's time to create control consist of RoundTextField + Error Message

Reply:
yes, I created a debt issue for this.